packages/esm-patient-chart-app/src/index.ts

```diff
@@ -32,16 +32,17 @@ function setupOpenMRS() {
   registerBreadcrumbs([
```

Comment:
It would probably be good to add some comments so that these keys get loaded, e.g.,

```
/**
 * t('patientBreadcrumb', 'Patient')
 * t'('Patient Summary Dashboard')
 */
```